src/components/PageBlocks/RecordListBase.vue

```diff
@@ -185,6 +185,8 @@
               >
                 {{ field.label }}
               </div>
+              <record-list-filter :selectedField="field" :fields="fields"
+              v-on:filter="onFilter" :recordListFilter.sync="recordListFilter"></record-list-filter>
```

Comment:
We will soon enable vue-recommended style plugin that will lint this. But for now, try to write every prop on a new line.
For a reference look at the b-btn component below